### PR TITLE
add citations for DEA products

### PR DESCRIPTION
**Pull request by CEKrause**
* [X ] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [X ] If required, update the [DEA Tech Alerts and Changelog](TechAlertsChangelog) and the [DEA Sandbox Updates banner](SandboxUpdatesBanner).

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

### CODE REVIEW

**Comment by CEKrause** [comment]
Pinging @robbibt as an FYI and in case you have any formatting requests. I have taken all the data citations from eCat, so should be non-controversial.

File: docs/data/product/dea-water-observations-statistics-landsat/_data.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ licence:
   link: https://creativecommons.org/licenses/by/4.0/
 
 citations:
-  data_citation: null
+  data_citation: "Mueller, N. 2022. Geoscience Australia Landsat Water Observation Statistics Collection 3. Geoscience Australia, Canberra. https://dx.doi.org/10.26186/146091"
   paper_citation: null
```
[the remark this answers is not in the document]
Comment (benji-glitsos-ga) [reply docs/data/product/dea-water-observations-statistics-landsat/_data.yaml:41]: Pinging @CEKrause

[the remark this answers is not in the document]
Comment (CEKrause) [reply docs/data/product/dea-water-observations-statistics-landsat/_data.yaml:41]: I ummd and ahhhd about that. I think we can go either way. I will add it in when I add in the geomedian citation. Benji has to update the back end to allow two paper citations.

[the remark this answers is not in the document]
Comment (robbibt) [reply docs/data/product/dea-water-observations-statistics-landsat/_data.yaml:41]: I just meant replace the current "null" paper citation with the Mueller paper, wouldn't need two paper citations (that paper includes WO statistics graphs so I think it'd be relevant to include )

**Pull request merged**